baseband/asp/payload.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+from ..vlbi_base.payload import VLBIPayloadBase
+from collections import namedtuple
+
+
+__all__ = ['ASPPayload']
+
+
+def decode_8bit(words):
```

Comment:
There's a version of `decode_8bit` that can just be imported from `vlbi_base.encoding`, but it's scaled to the "8-bit 1 sigma" value.  Is the lack of scaling here deliberate?  If not, we can just import from `encoding`.

Reply:
we do the same for DADA, GUPPI, and GSB, so keeping for now. May want to combine those at some point...